### DISCUSSION TITLE
background opacity stops working with multiple panes

### DISCUSSION THREAD

**Discussion by ackstorm23** [wezterm/wezterm#1522] [Q&A] [answered]
my background opacity works fine with a single pane, but as soon as I split panes my background opacity is lost and I have a solid background color.

how do I ensure the opacity is retained when splitting panes?

**Comment by wez** [comment, marked as answer]
Sounds like https://github.com/wez/wezterm/issues/1459
What version are you running?

**Comment by ackstorm23** [comment]
**20211205-192649-672c1cc1**

I didn't search for 'tiling' or 'splitting' so I seem to have missed the existing bug report, sorry!

I'll update to latest.

**Comment by ackstorm23** [comment]
Confirmed working great on 20220101-133340-7edc5b5a
